Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.13-alpine as builder
+MAINTAINER siddontang
+
+RUN apk add --no-cache \
+    git make wget
+
+RUN wget http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz && \
+    tar -xzf tidb-enterprise-tools-latest-linux-amd64.tar.gz && \
+    mv tidb-enterprise-tools-latest-linux-amd64 /opt/tools
+
+RUN mkdir -p /go/src/github.com/pingcap/dm
+WORKDIR /go/src/github.com/pingcap/dm
+
+# Cache dependencies
+COPY go.mod .
+COPY go.sum .
+
+RUN GO111MODULE=on go mod download
+
+COPY . .
+
+RUN make dm-worker dm-master dm-tracer dmctl
+
+FROM alpine:3.5
```

Comment:
How about using `alpine:3.10` (resued from  `golang:1.13-alpine`, ref  https://github.com/docker-library/golang/blob/f30f0428221b94c7dcb414ebdcea83106df20897/1.13/alpine3.10/Dockerfile)

Reply:
@iamxy 

which Apline version do you use now?

Reply:
I prefer `alpine:3.10`.

Reply:
The current base image used by our official images is pingcap/alpine-glibc. It's still 3.8 not 3.10.
I think we need to update the base image to 3.10. /cc @qrr1995

Reply:
so let's use 3.10 here.

Reply:
The `pingcap/alpine-glibc:latest` image  has been updated to alpine-3.10 now, just check it   with 
```
$ docker pull pingcap/alpine-glibc
$ docker run pingcap/alpine-glibc cat /etc/alpine-release
```